src/Header/Header.less

```diff
@@ -172,6 +172,7 @@
 					color: @moon-input-header-placeholder-color;
 					margin-top: 12px;
 					line-height: 1.25em;
+					opacity: 1;
```

Comment:
Could we instead be specific to FireFox only?
`
::-moz-placeholder {
  opacity: 1;
}
`